src/workbooks/governance/workbook.json

```diff
@@ -122,6 +122,18 @@
             },
             "name": "RC_Menu"
           },
+          {
+            "type": 1,
+            "content": {
+              "json": "## Azure Governance Workbook\r\n### Change Log\r\n\r\n|Version|Date|Description|\r\n|---|---|---|\r\n|v1.0.0| 12/07/2023 | Initial Release\r\n|v1.1.0| 13/07/2023 |-Added dynamic filter for Cost Management tab<br>-Query fixed for Tag Explorer<br>-Fixed typo in all tab notes"
```

Comment:
Should the changelog be in the workbook or in GitHub? We at least need something basic in GitHub, but other than that I'll leave it to you.

Reply:
You are right, I will remove it thanks.